src/locales/en.yaml

```
@@ -83,6 +83,8 @@ app:
       uploaded: Uploaded
       view_section_documentation: >-
         View '%{section}' documentation
+      show_hidden_files: Show hidden files
+      show_klipper_backup_files: Show klipper backup files
```

While the capitalization of Klipper isn't consistent across the repo, the translations all seem to use it in upper case
```suggestion
      show_klipper_backup_files: Show Klipper backup files
```